wallettemplate/build.gradle

```diff
@@ -12,11 +20,16 @@ dependencies {
     implementation 'org.slf4j:slf4j-jdk14:1.7.25'
 }
 
-sourceCompatibility = 1.8
+javafx {
+    modules = [ 'javafx.controls', 'javafx.fxml' ]
+}
+
+sourceCompatibility = 1.11
 compileJava.options.encoding = 'UTF-8'
 compileTestJava.options.encoding = 'UTF-8'
 
-task wallet_template(type: JavaExec) {
-    main = 'wallettemplate.Main'
-    classpath = sourceSets.main.runtimeClasspath
+mainClassName = 'wallettemplate.Main'
+
+// task wallet_template and can be replaced with the 'run' task
```

Comment:
This migration could actually be its own PR, since it's an independent idea that can be discussed and merged independently.

Reply:
Actually, I think it's a byproduct of the OpenJFX plugin which uses/requires the Gradle 'application' plugin.

Reply:
Ah ok, didn't know. Thanks.